apps/mobile/src/libs/components/safe-area-screen-wrapper/safe-area-screen-wrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {
+	SafeAreaView,
+	useSafeAreaInsets,
+} from "react-native-safe-area-context";
+
+type Props = {
+	children: React.ReactNode;
+};
+
+const SafeAreaScreenWrapper: React.FC<Props> = ({ children }) => {
+	const insets = useSafeAreaInsets();
+
+	return (
+		<SafeAreaView
+			style={{ flex: 1, paddingTop: insets.top, paddingBottom: insets.bottom }}
```

Comment:
Let's add the `edges` ('top', 'button', 'right', 'left') prop to the wrapper to be able to set paddings explicitly.